common/cuda_hip/matrix/batch_dense_kernels.hpp.inc

```diff
@@ -35,6 +35,109 @@ template <typename ValueType>
 using BatchEntry = gko::batch_dense::BatchEntry<ValueType>;
 
 
+template <typename ValueType>
+__device__ __forceinline__ void single_spmv_kernel(
```

Comment:
Maybe rename to `singke_apply_kernel` ? It is not sparse in this case :)

Reply:
Yeah, I noticed that, but the name needs to be the same across all matrix formats for the batch solvers to work 😅 Maybe we change it to `single_apply_kernel` everywhere, though for free functions that's not as descriptive. Maybe `single_matvec_kernel`?

Reply:
Ah, yeah. True. Maybe we can rename it to `single_apply_kernel` everywhere.